lib/project_types/theme/tasks/ensure_themekit_installed.rb

```diff
@@ -0,0 +1,36 @@
+module Theme
+  module Tasks
+    class EnsureThemekitInstalled < ShopifyCli::Task
+      FILENAME = File.join(ShopifyCli::CACHE_DIR, "themekit")
+      URL = 'https://shopify-themekit.s3.amazonaws.com/releases/latest.json'
+      OSMAP = {
+        mac: 'darwin-amd64',
```

Comment:
I think we'll also need to handle the i386 binaries here?

Reply:
Both ngrok and Heroku use only 64-bit downloads, so I will do the same here unless issues arise.